src/main/java/uk/gov/hmcts/ccd/endpoint/std/DraftsEndpoint.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.ccd.endpoint.std;
+
+import io.swagger.annotations.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.ccd.domain.model.std.CaseDataContent;
+import uk.gov.hmcts.ccd.domain.model.draft.Draft;
+import uk.gov.hmcts.ccd.domain.service.createdraft.SaveDraftOperation;
+
+@RestController
+@RequestMapping(path = "/",
+    consumes = MediaType.APPLICATION_JSON_VALUE,
+    produces = MediaType.APPLICATION_JSON_VALUE)
+@Api(value = "/", description = "Drafts API")
+public class DraftsEndpoint {
+    private final SaveDraftOperation saveDraftOperation;
+
+    @Autowired
+    public DraftsEndpoint(@Qualifier("default") final SaveDraftOperation saveDraftOperation) {
+        this.saveDraftOperation = saveDraftOperation;
+    }
+
+    @RequestMapping(value = "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/event-trigger/{etid}/drafts", method = RequestMethod.POST)
```

Comment:
Given the Drafts would be listed per case type, not per event, I wonder if it makes sense to create them against an event-specific resource.

```POST /caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/drafts``` would feel more appropriate, however it would require passing the event ID as part of the body.

Reply:
The endpoint was built this way so we store the draft with all metadata required. This metadata is needed to build CaseView and populate the event trigger dropdown on case view.